util/src/main/java/io/zeebe/util/buffer/BufferUtil.java

```diff
@@ -245,6 +245,33 @@ public static boolean startsWith(final DirectBuffer buffer, final DirectBuffer p
     return true;
   }
 
+  /**
+   * Performs byte wise comparison of a given buffer and a prefix.
+   *
+   * @param prefix the prefix to look for
+   * @param content the buffer to check against
+   * @return true if buffer starts with the all bytes contained in prefix
+   */
+  public static boolean startsWith(
```

Comment:
We have two `startsWith` implementation, we can probably have one reuse the other (I think it's just overloading, one is with `byte[]` and the other with `DirectBuffer`).